### PR TITLE
Support all artifact result outputs

### PR DESCRIPTION
**Pull request by joejstuart**
This change supports all chains artifacts outputs. It uses all outputs to identify the build task.

https://issues.redhat.com/browse/EC-590

### CODE REVIEW

**Comment by joejstuart** [comment]
Setting as draft while adding functionality for multiple results.

File: policy/lib/tekton/task_results.rego [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package lib.tkn
+
+import rego.v1
+
+# handle the output artifacts from Tekton Chains
+# https://tekton.dev/docs/chains/slsa-provenance/#output-artifacts
+
+# returns the value of a task result with name IMAGE_URL
+task_result_url(task) := value if {
```
Comment (zregvart) [inline policy/lib/tekton/task_results.rego:9]: Might be a bit more straightforward if this was named `task_result_artifact_url`?

**Review by zregvart — APPROVED** [review]
LGTM, make sure to run `make generate-docs` to regenerate the documentation

**Comment by joejstuart** [comment]
@zregvart I added a commit in case you wanna take another look. https://github.com/enterprise-contract/ec-policies/pull/1013/commits/ddbcb6680f5db84abdd5f8e1a2ce51a255afc151

**Review by zregvart — APPROVED** [review]
🚀

**Pull request merged**